tests/ampi/migration/test.C

```diff
@@ -1,63 +1,92 @@
 /**
- * AMPI Thread Migration Test
- * Migrate the rank 1 AMPI process from node to node in order.
+ * AMPI Migration Test:
+ * Migrate AMPI rank 1 from PE to PE in order.
  */
 #include <stdio.h>
 #include <stdlib.h>
+#include <vector>
+#include <cassert>
 #include "mpi.h"
-#include "charm.h" /* For CkAbort */
 
+int get_my_pe() {
+  int flag;
+  int *my_wth;
+  MPI_Comm_get_attr(MPI_COMM_WORLD, AMPI_MY_WTH, &my_wth, &flag);
+  assert(flag == 1);
```

Comment:
Is it okay to call assert from MPI programs, or is it more correct to call MPI_Abort if the test fails?

Reply:
MPI_Abort is better / more graceful. Changed.